host/4/dotnet-build.yml

```diff
@@ -1,3 +1,4 @@
+# TODO update me
```

Comment:
Waiting on this PR: https://github.com/Azure/azure-functions-docker/pull/1036/files